### ISSUE TITLE
Please add the possibility to specify the color of the badge

### ISSUE THREAD

**Issue by sblantipodi** [gilmaimon/ArduBadge#14]
As title.
Please add the

**Comment by bblanchon** [comment]
Agreed! We need to be able to change the color to match the other badges.
See the new PlatformIO badge for example: [![PlatformIO Registry](https://badges.registry.platformio.org/packages/bblanchon/library/ArduinoJson.svg)](https://registry.platformio.org/packages/libraries/bblanchon/ArduinoJson).
Also, we need to be able to remove the name of the package.
